scripts/jenkins-build.bat

```diff
@@ -0,0 +1,65 @@
+@echo off
+REM Build with Jenkins CI.  This script is intended for use with the OME
+REM Jenkins CI infrastructure, though it may be useful as an example for
+REM how one might use various cmake options.
+
+REM So that statements like "set" work inside nested conditionals
+setlocal ENABLEDELAYEDEXPANSION
+setlocal ENABLEEXTENSIONS
+
+set "script=%0"
+
+set "workspace=%~dp0\.."
```

Comment:
Using the layout from this PR description (`source/scripts/jenkins-build.bat` under the Jenkins workspace), this set of calls does not detect the correct workspace - see https://ci.openmicroscopy.org/view/Files/job/OME-FILES-performance/6/console.

Looking at the corresponding [ome-cmake-superbuild script](https://github.com/ome/ome-cmake-superbuild/blob/master/scripts/jenkins-build.bat), the difference is that the CI job uses the `-w %WORKSPACE%` to override this logic and pass the workspace environment variable.